float32 x
float32 y
string state
#定义了msg的格式：x、y表示GPS定位的坐标，state表示工作状态
#编译之后会生成一个.h文件
